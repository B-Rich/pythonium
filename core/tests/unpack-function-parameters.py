def func(a, b, c, d=1, f=2, *args, **kwargs):
    print(a)
    a = 1
    print(a, b, c, d, f)
    print(args[0])
    print(args[1])
    print(kwargs)

kwargs = {"foo": "bar"}
func(42, 43, 44, 45, 46, 100, 102, key="value", **kwargs)
