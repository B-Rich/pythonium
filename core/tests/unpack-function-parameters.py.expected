42
1 43 44 45 46
100
102
{ foo: 'bar', key: 'value' }
